gradle.properties

```diff
@@ -2,14 +2,15 @@
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
+org.gradle.java.installations.fromEnv=JAVA_17_HOME
 
 modID=hexcasting
 modName=Hex Casting
 
 jetbrainsAnnotationsVersion=23.0.0
 
 minecraftVersion=1.20.1
-kotlinVersion=1.7.20
+kotlinVersion=1.9.0
```

Comment:
Why is Kotlin being updated? (also, if we're doing that, should KFF be updated as well?)

Reply:
hm
No idea why I did this one